src/symposium-speakers/symposium-speakers.html

```diff
@@ -71,6 +71,19 @@
     </section>
 
     <section>
+      <symposium-speaker name="Jouke Waleson (Mendix)" title="Building a product with just enough security" image="/2017/images-optimized/speakers/jouke_waleson.jpg">
+        <b>Abstract</b><br>
+        "Security is our number one priority". Many companies say something like this but it's mostly marketing rhetoric: no company will (or should) drop all work for a minor security issue. Product Management is about determining the priorities and the direction for a product, in a rapidly evolving market under heavy constraints. It often feels like juggling while sinking in quicksand, surrounded by predators. At Mendix, this is my job. My product is the Mendix Cloud, where our customers run mission critical apps which often contain highly sensitive data. In this talk I'll show you three things:<br>
+        - how our security priorities shifted over time as we landed our first enterprise customers;<br>
```

Comment:
Would be better to use `<ul>` but if you can't fix that nicely let's leave like this.